src/mme-app/actionHandlers/attachActionHandlers.cpp

```diff
@@ -642,6 +641,8 @@ ActStatus ActionHandlers::process_esm_info_resp(SM::ControlBlock& cb)
 	const struct esm_resp_Q_msg &esm_res =s1_msg_data->msg_data.esm_resp_Q_msg_m;
 
 	procedure_p->setRequestedApn(Apn_name(esm_res.apn));
+	ue_ctxt->setUpLnkSeqNo(ue_ctxt->getUpLnkSeqNo()+1);
```

Comment:
Confused. Can we not just call increment api ? That would be much easier. Also not sure what we are just incrementing it, can you please give logic.

Reply:
Currently only getter and setter is supported for datablocks. Hence doing it this way.